cogment_verse/web/web_app/src/pages/Play.jsx

```diff
@@ -59,7 +59,7 @@ const Play = () => {
   const redirectToPlayAny = useCallback(() => navigate(".."), [navigate]);
 
   return (
-    <div className="max-w-screen-md mx-auto min-h-screen justify-center">
```

Comment:
Very tiny changed that fixed a very annoying detail of the web app for a long time. Playing a game with up and down arrows would slightly scroll up and down the screen during a trial, even if the render and instructions perfectly fitted vertically within the window.